app/static/index.js

```diff
@@ -16,13 +16,24 @@ var evaluate = function evaluate(js) {
         // which will be returned to PSCi.
         buffer.push(s);
     };
-    // Replace any require(...) statements with lookups on the PSCI object.
+    // Replace any require and import statements with lookups on the PSCI object
+    // and export statements with assignments to module.exports.
     var replaced = js.replace(/require\("[^"]*"\)/g, function(s) {
         return "PSCI['" + s.split('/')[1] + "']";
+    }).replace(/import \* as ([^\s]+) from "([^"]*)"/g, function (_, as, from) {
+        return "var " + as + " = PSCI['" + from.split('/')[1] + "']";
+    }).replace(/export \{([^}]+)\} from "\.\/foreign\.js";?/g, function (_, exports) {
+        return exports.replace(/^\s*([^,\s]+),?\s*$/gm, function (_, exported) {
+            return "module.exports." + exported + " = $foreign." + exported + ";";
+        });
+    }).replace(/export \{([^}]+)\};?/g, function (_, exports) {
+        return exports.replace(/^\s*([^,\s]+)(?: as ([^\s]+))?,?\s*$/gm, function (_, exported, as) {
+            return "module.exports." + (as || exported) + " = " + exported + ";";
+        });
```

Comment:
Should these regexes account for extra spaces? For example
```js
import * as f from "fs"; // normal spacing
import  * as f from "fs"; // typo space
```
The `require` regex on which these are likely based did not need to account for extra whitespace issues.

Reply:
afaik this is in the repl in browser feature, which hasn't been updated yet.

Reply:
Ah, I see. This code only runs when `purs repl --port somePort` is called. I've always just used `purs repl`, which is always using the Node backend. 

> which hasn't been updated yet.

But shouldn't this be fixed here? Or does that mess up tests and hence why it hasn't yet?

Reply:
It's not clear to me what the resolution was here. Do we need to note this in the release? Will it be updated or will it be removed? Will that happen in a subsequent PR?

Reply:
Wasn't the idea to remove the browser repl in a subsequent PR?

Reply:
I'm fine with removing it, but it wasn't clear to me there was any consensus here with the core team. Did I miss that?

Reply:
I've missed a few meetings, but I don't think there has been consensus yet as I don't think this has come up. I'm up for removing it though because I've never used the browser version. If I want a REPL-like experience, I usually just use Try PureScript because there's a small bug in one part of my code and starting a REPL just to debug that one function isn't worth it to me.